### PR TITLE
Add `data` to UserList class in py2

### LINKED ISSUE #1759: Missing data attribute in UserDict, UserList and UserString
All three classes define the `data` attribute and it's definition is missing in typeshed.

### PR DESCRIPTION
**Pull request by CraftSpider**
Closes #1759 

Very small issue, as the `data` attributes for both other classes were added along the way at some point

### CODE REVIEW

File: stdlib/2/UserList.pyi [hunk 1]
```diff
@@ -1,9 +1,10 @@
-from typing import Iterable, MutableSequence, TypeVar, Union, overload
+from typing import Iterable, MutableSequence, TypeVar, Union, overload, Sequence
 
 _T = TypeVar("_T")
 _S = TypeVar("_S")
 
 class UserList(MutableSequence[_T]):
+    data: Sequence[_T]
```
Comment (srittau) [inline stdlib/2/UserList.pyi]: This is actually always a real `List`.

Comment (CraftSpider) [reply stdlib/2/UserList.pyi]: I was following the example of UserDict, which sets its data type to `Mapping[...]`. Looking at the code, it is guaranteed to always be a Dict. Should I change that as well?

Comment (srittau) [reply stdlib/2/UserList.pyi]: That would be great!

Comment (CraftSpider) [reply stdlib/2/UserList.pyi]: Done

**Pull request merged**